CI/travis.before_install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Before install on linux.
```

Comment:
Might I suggest leaving the conditional here and a place-holder

    echo Before install on linux - currently no travis.linux.before_install.sh present or expected

this is so that when inspecting the Travis build logs we have a reminder of what is not happening (as expected) on Linux compared to on MacOs - and so we have the framework in place should such shenanigans become necessary again?